src/daemon/Display.cpp

```diff
@@ -161,7 +173,9 @@ namespace SDDM {
             }
             // It might be the case that we are trying a tty that has been taken over by a
             // different process. In such a case, switch back to the initial one and try again.
-            VirtualTerminal::jumpToVt(SDDM_INITIAL_VT, true);
+            if (seat()->canTty()) {
```

Comment:
This should never be reached

Reply:
OK, yeah that seems to check out it only gets triggered on ttyFailed which only gets triggered when takeControl is true and it fails, and `vtFd > 0)` right?

I'm seeing that now. Done locally